src/components/Dropdown/Dropdown.js

```diff
@@ -20,8 +20,8 @@ const Item = styled.div`
     props.disabled &&
     `
     cursor: not-allowed;
-    color: ${props.theme.colors.gray200};
-    background-color: ${props.theme.colors.white};
+    color: ${props.theme.colors.gray600};
```

Comment:
gray600 is color of placeholder text in input component when disabled. Not sure if it should be rgb(0, 0, 0) or something else

Reply:
`gray600` seems good!